tools/clang/include/clang/Frontend/CodeGenOptions.h

```diff
@@ -42,44 +43,40 @@ class CodeGenOptionsBase {
 class CodeGenOptions : public CodeGenOptionsBase {
 public:
   enum InliningMethod {
-    NoInlining,         // Perform no inlining whatsoever.
-    NormalInlining,     // Use the standard function inlining pass.
-    OnlyAlwaysInlining  // Only run the always inlining pass.
+    NoInlining,        // Perform no inlining whatsoever.
```

Comment:
the clang-format extension keeps changing the spacing in this file against my will :/

Reply:
We'd better to undo this and keep the original LLVM source code untouched. Applies to following cases too.